task/aws-ecr-login/0.1/samples/test-task.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       type: string
   steps:
     - name: pull-and-push
-      image: quay.io/buildah/stable:v1.14.8
+      image: quay.io/buildah/stable
```

Comment:
we do want to have the version in there (And everywhere), isnt that the best practices for images or something with the tag ? cc @vdemeester

Reply:
Indeed then probably we need to fix [here](https://github.com/tektoncd/catalog/blob/master/task/s2i/0.1/s2i.yaml#L64) as well. :slightly_smiling_face: